modules/replicator/src/testFixtures/java/org/apache/ignite/internal/raft/client/AbstractTopologyAwareGroupServiceTest.java

```diff
@@ -215,7 +215,7 @@ private IgniteBiTuple<TopologyAwareRaftGroupService, TopologyAwareRaftGroupServi
 
         assertNotNull(raftClient);
 
-        raftClient.refreshLeader().get();
+        raftClient.refreshLeader().get(WAIT_TIMEOUT_MILLIS, TimeUnit.MILLISECONDS);
```

Comment:
Why do we need external timeout here? I mean that refreshLeader internally has time bounded limitations:
```
            if (currentTimeMillis() >= stopTime) {
                fut.completeExceptionally(
                        new TimeoutException(format("Send with retry timed out [retryCount = {}, groupId = {}].", retryCount, groupId)));

                return;
            }
```
?

Reply:
Didn't noticed internal limitation, just was triggered with `get()` without a timeout. Removed.

Reply:
Please check other places, e.g. on line 237.